OrbitGl/TimeGraph.cpp

```diff
@@ -214,6 +215,26 @@ void TimeGraph::PanTime(int a_InitialX, int a_CurrentX, int a_Width,
   NeedsUpdate();
 }
 
+//-----------------------------------------------------------------------------
+void TimeGraph::VerticalAlign(const TextBox* a_TextBox) {
+  if (!a_TextBox) return;
```

Comment:
If `a_TextBox` is a nullptr, then it's clearly a programming error. I would recommand to use `CHECK(a_TextBox != nullptr)` in that case.

Reply:
Yeah, you are right. Done!